src/software/ai/hl/stp/action/intercept_ball_action_test.cpp

```diff
@@ -6,6 +6,8 @@
 #include "software/test_util/test_util.h"
 #include "software/world/ball.h"
 
+// TODO (Issue #1644): refactor and reenable these tests
+/*
```

Comment:
Sorry if this is explained elsewhere, why are these tests commented out? Do we still have coverage on these actions?

Reply:
This PR makes *Intent not extend *Primitive, so the functions like getDestination() don't work anymore. However, after this PR we're still using PrimitiveParamMsg, so to make these tests work, I'd have to change all the tests to look at individual params. After #1643, I can reenable the tests by accessing the fields in the new proto, which are much more user friendly